InMobiAdapter/src/main/java/com/chartboost/mediation/inmobiadapter/InMobiAdapter.kt

```diff
@@ -134,6 +139,7 @@ class InMobiAdapter : PartnerAdapter {
                             continuation.resume(result)
                         }
                     }
+                    logLevel = InMobiSdk.LogLevel.DEBUG
```

Comment:
I don't think it's a good idea to do this here. Do it from Canary instead. Else this will override the user-supplied log level all the time at init

Reply:
Oops. Good call. Was debugging. 👍🏼

Reply:
Applied on https://github.com/ChartBoost/chartboost-mediation-android-adapter-inmobi/pull/47/commits/2bbeff1a52ef6d8a91ac18acf85385666cffcd78